raiden/network/transport/matrix/transport.py

```diff
@@ -354,8 +357,10 @@ def start(  # type: ignore
             raise RuntimeError(f"{self!r} already started")
         self._stop_event.clear()
         self._raiden_service = raiden_service
+        if self._raiden_service.wal:
+            self.storage = self._raiden_service.wal.storage
```

Comment:
It seems to me that the convention in the matrix transport is to prefix member variables with `_`. Could you please change this?